test/staking/Integration.ts

```diff
@@ -523,7 +546,7 @@ describe("Integration", function () {
 
       await receiverDeposit(pond, receiverStaking, receiver1, minPondToUseByReceiver.div(2));
       await receiverDeposit(pond, receiverStaking, receiver2, minPondToUseByReceiver);
-      const fraction = BN.from(10).pow(18);
+      const fraction = BN.from(2).pow(16).sub(1);
```

Comment:
Probably declare a constant somewhere, easier that changing everywhere na

Reply:
done